src/playground/sample_data_ingestion/sample_data_ingestor/sample_data_ingestor.py

```diff
@@ -0,0 +1,345 @@
+from datetime import datetime, timedelta
+from shutil import copyfileobj
```

Comment:
could you please follow the same folder instruction which I suggested for sample tool? Thanks

Reply:
Done, folder structure has been refactored